sechub-pds-solutions/iac/09-compute-image-tag.sh

```diff
@@ -12,6 +12,10 @@ else
   VERSION_TAG="`date +%Y-%m-%d`"
 fi
 
+if [[ -n "$KICS_VERSION" ]] ; then
```

Comment:
Suggest to leave out this section because in future the container may contain other tools than Kics.